script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -782,16 +788,17 @@ If lines represent the death of life,
 [sha:df8c498b97cf72836e228c3f1dfc166cd82e0a56]{
 -- Page 704, Offset 10566.
 -- どのような不死身、どのような怪物であれ一刺しで“殺す”、この世でもっともおぞましい、宇宙の終わりに他ならない。
-Any immortal being,
- any monster,
- can be 'killed' with a single strike.
+//The ambiguous pronoun in the first sentence is a bit annoying, so I considered a construction like "No matter how unkillable an immortal――no matter how terrible a monster..." but it's sort of long, and it duplicates kill.
+No matter how immortal――
+no matter how monstrous,
+ all it would take to kill them is a single stab.
```

Comment:
Feel like this doesn't flow great from the prev line, thoughts on
> A point that can bring about the death of anyone―no matter how immortal, no matter how monstrous――with a single stab.

Reply:
It certainly needs the extra pronoun here, true.
Looking at it again, it might even need some extra stuff in the second clause.

>No matter how immortal――
>no matter how monstrous,
>a single stab in that point is enough to kill anything,
>It is the most dreadful thing in the world,
>nothing less than the end of the universe.

That way the dramatic structure is preserved.